.github/workflows/ci-failures-report.yml

```diff
@@ -36,3 +36,4 @@ jobs:
         slack_top_n_failures ${n} "operator-e2e" ":operator: Top ${n} Operator E2E Test failures in this and previous week" "${{github.event.inputs.test}}"
         slack_top_n_failures ${n} "ui-e2e" ":computer: Top ${n} UI E2E Test failures in this and previous week" "${{github.event.inputs.test}}"
         slack_top_n_failures ${n} "nongroovy-e2e" ":go: Top ${n} NonGroovy E2E Test failures in this and previous week" "${{github.event.inputs.test}}"
+        slack_top_n_failures ${n} "go" ":github: Top ${n} unit test failures in this and previous week" "${{github.event.inputs.test}}"
```

Comment:
I'm somewhat afraid `CONTAINS_SUBSTR(ShortJobName, "go")` might trigger some unexpected matches in the future as we introduce new jobs :-(
I haven't looked at the values we have currently, but maybe using something like `"-go-"` or `"-go"` help with avoiding that? 🤔

Reply:
we have `go` and `go-postgres`
If we add something else we could see it and fix.